spec/calendar.html

```diff
@@ -879,7 +867,7 @@ <h1>Temporal.Calendar.prototype.toJSON ( )</h1>
       </p>
       <emu-alg>
         1. Let _calendar_ be the *this* value.
-        1. Return ? CalendarToString(_calendar_).
+        1. Return ? ToString(_calendar_).
```

Comment:
a reasonable alternative would be to Call %Temporal.Calendar.toString%, but this way plays better with subclassing i suppose.

Reply:
This way does an additional observable Get(calendar, @@toPrimitive) but I think that's OK?

Reply:
Yes, either a Symbol.toPrimitive or a toString lookup would be required to invoke subclass code.